### PR TITLE
Use small-1.x influxdb service plan in prod

### PR DESCRIPTION
**Pull request by vigneshmsft**
Use `small-1.x` influxdb service plan in prod

### CODE REVIEW

File: monitoring/variables.tf [hunk 1]
```diff
@@ -20,4 +20,5 @@ locals {
   paas_api_url             = "https://api.london.cloud.service.gov.uk"
   monitoring_space_name    = "bat-${var.monitoring_env}"
   monitoring_instance_name = var.monitoring_env == "prod" ? "bat" : "bat-${var.monitoring_env}"
+  influxdb_service_plan    = var.monitoring_env == "prod" ? "small-1_x" : "tiny-1_x"
```
Comment (saliceti) [inline monitoring/variables.tf:23]: Can we move this to a new tfvars file instead?
monitoring_instance_name could go as well

**Comment by saliceti** [comment]
[API migration - Upgrade influxDB](https://trello.com/c/21ChKdJr/1194-api-migration-upgrade-influxdb)

File: monitoring/variables.tf (hunk 1 above)
Comment (vigneshmsft) [reply monitoring/variables.tf]: @saliceti done

**Review by saliceti — APPROVED** [review]

File: monitoring/variables.tf (hunk 1 above)
Comment (saliceti) [reply monitoring/variables.tf]: Thanks 👍

**Pull request merged**